scripts.py

```diff
@@ -163,48 +156,6 @@ def q(self):
         )
 
 
-class UpdateStaffPicksScript(Script):
```

Comment:
Flamethrowers cast a wide net I suppose :fire:

Reply:
This one was removed because it used the old configuration settings to configure the script: 
`Configuration.integration_url(Configuration.STAFF_PICKS_INTEGRATION, True)`

Since we never setup this config file, it seemed like this script was dead as well.